static/src/javascripts/projects/common/modules/commercial/adblock-ask.js

```diff
@@ -12,20 +12,9 @@ const supportUrl = `${supportSubscribeDigitalURL()}?acquisitionData=%7B%22compon
 
 const askHtml = `
 <div class="contributions__adblock">
-    <div class="contributions__adblock-content">
-        <div class="contributions__adblock-header">
-            <h2>
-                Read The<br>
-                Guardian without<br>
-                interruption on all<br>
-                your devices
-            </h2>
-        </div>
-        <a class="contributions__adblock-button" href="${supportUrl}">
-            <span class="component-button__content">Subscribe now</span>
-        </a>
-        <img src="https://media.guim.co.uk/b437f809d9fa4c72336ccbead1730b6bb0965239/0_0_432_503/432.png" class="contributions__adblock-image" alt="" />
-    </div>
+    <a href="${supportUrl}">
+        <img src="https://media.guim.co.uk/3edc720c6210f873c7e75df4c46bea75615401d8/0_0_300_250/300.jpg" alt="" />
```

Comment:
could this be a png or gif?

Reply:
Unfortunately, no. I discussed with @i-hardy, and this is the only source file available.

This PR needs to be merged very shortly, too.

Reply:
Yes, I would love to make this a little shinier but the timeline is unfortunately pretty tight. I think we're going to be reviewing the shady pie in the near future so hopefully there will be a chance to either significantly improve on this or get rid entirely.

Reply:
See below.